infra/src/main/kotlin/fi/fta/geoviite/infra/geocoding/AddressPointsCache.kt

```diff
@@ -56,8 +56,9 @@ class AddressPointsCache(
         }
     }
 
-    /** This is for caching address points. Please don't call this directly if possible, please
```

Comment:
Branchissa on mukana jonkunmoinen kasa warneja/detekt-nillityksiä poistavia siistimisiä. Ikävää että ne on taas näin ison pullarin seassa, mutta koitin varmistella jälkiäni sillä warnit osoitti monesti paikkoihin joita olin unohtanut -> noiden tarkastus on puolestaan paljon helpompaa kun joka paikassa ei ole turhaa formatointinillitystä.

Ei kannattane niitä kummemmin reviewata, merkkaan tänne ne kohdat joissa muuttuu jotain muutakin.